tests/cypress/integration/actions_projects/base_actions_project_task_user.js

```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
             cy.openTask(taskName.firstTask);
             cy.get('.cvat-constructor-viewer').should('not.exist');
         });
-        it('Logout first user, register second user, logout.', () => {
+        it('Logout first user, register second user, tries to create project and logout.', () => {
             cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
```

Comment:
Is this window expected to be visible after each logout? Maybe put it inside logout?

Reply:
These actions moved in cy.logout().